doc/cookbook.rst

```diff
@@ -14,3 +14,23 @@ Slash offers a hook called ``tests_loaded`` which can be used, among else, to co
 	       test.__slash__.set_sort_key(index)
 
 The above code is best placed in a ``slashconf.py`` file at the root of your test repository.
+
+
+Adding Multiple Log Files to the Current Result
```

Comment:
I think this would be more complete along with an example of how to add multiple log files to a single test as well..